projects/mno-frontend-lib/src/lib/_components/login-box/login-box.component.ts

```diff
@@ -24,15 +29,37 @@ export class LoginBoxComponent implements OnInit {
 
   public login() {
     this.loading = true
-    this.auth.login(this.email, this.password).subscribe(
-      (u: User) => {
-        this.loggedIn.emit(u)
-        this.loading = false
+    this.auth.login(this.email, this.password).pipe(
+      catchError((err) => {
+        this.loading = false;
```

Comment:
If you just want to set loading to false on error, I think you can just use `finalize(() => this.loading = false)`. Oh, you seem to be doing that below, am I missing something?

Reply:
I want it to stop loading only on error, not on success (for the login action, not signup).
Otherwise, we can see the button stops loading for a split second before the redirection happens, which does not look good... To be improved later